pkg/agent/cniserver/pod_configuration.go

```diff
@@ -521,28 +522,11 @@ func (pc *podConfigurator) connectInterceptedInterface(
 	if err != nil {
 		return err
 	}
-
-	containerIface := &current.Interface{}
-	intf, err := util.GetNSDevInterface(containerNetNS, containerIFDev)
-	if err != nil {
-		return fmt.Errorf("connectInterceptedInterface failed to get veth info: %w", err)
-	}
-	containerIface.Name = containerIFDev
-	containerIface.Sandbox = sandbox
-	containerIface.Mac = intf.HardwareAddr.String()
-
-	// Setup dev in host ns.
-	hostIface := &current.Interface{}
-	intf, br, err := util.GetNSPeerDevBridge(containerNetNS, containerIFDev)
+	containerIface, hostIface, err := pc.ifConfigurator.getInterceptedInterfaces(sandbox, containerNetNS, containerIFDev)
```

Comment:
This sounds strange to me to refactor code for unsupported funcs like connectInterceptedInterface and disconnectInterceptedInterface. How about move these two funcs to interface_configuration_os.go?

Reply:
Humm.. maybe then we have problems with other PodConfigurator references like routeClient, connectInterfaceToOVS()?

Reply:
Correct. If I move the calling functions in interface_configuration.go, routeClient is also needed to be passed to interfaceconfigurator. Then the struct dependencies might be confused.

Reply:
Ok.